redirects-list.json

```diff
@@ -1,5 +1,6 @@
 [
   "^https://www\\.cml\\.dev(.*)?                                           https://cml.dev$1",
+  "^/chat(/.*)?$                                                           https://discord.gg/ucXJVerWee",
```

Comment:
@jendefig just checking is this the correct link for CML?

Reply:
Yes, it should be the one with 'Wee" because that was the shorter version [per your request](https://www.notion.so/iterative/Discord-Link-Sources-aabf174e81a74b489fce2472db796ebf).  The OG version from Elle with the history, was https://discordapp.com/invite/bzA6uY7, not as you typed above, but with the same beginning as the DVC website has.  

I tried the link above and it takes me to the server because I have Discord open on my computer to the server.  But if I tried the link in incognito, it sent me to set up an account, and when I signed in, it sent me to my account, not the server.

Reply:
> I tried the link above and it takes me to the server because I have Discord open on my computer to the server. But if I tried the link in incognito, it sent me to set up an account, and when I signed in, it sent me to my account, not the server.

I think that's normal behavior for discord. I tested it on the dvc link and got the same thing. But if you already have an account (logged in or not logged in) you will go to the cml channel :)

Reply:
```suggestion
  "^/chat(/.*)?$                                                           https://discord.gg/bzA6uY7",
```